extensions/amp/bundle/src/test/java/com/adobe/cq/wcm/core/extensions/amp/internal/AmpModeForwardFilterTest.java

```diff
@@ -164,5 +168,32 @@ void testFilteringWithSelectorWithAmpMode() throws IOException, ServletException
         spyAmpModeForwardFilter.doFilter(context.request(), context.response(), filterChain);
         verify(spyAmpModeForwardFilter, atLeastOnce()).doFilter(eq(context.request()), eq(context.response()) , eq(filterChain));
     }
+    
+    
+    
+    @Test
+    void testWithDisabledFilter() throws IOException, ServletException {
+    	AmpModeForwardFilter filter = Mockito.spy(new AmpModeForwardFilter(new AmpModeForwardFilter.Config() {
```

Comment:
I wanted to call the only public constructor (which is also used by OSGI) at least once. Otherwise if this constructor is removed (for whatever reason) no test will fail.